packages/Webkul/Velocity/src/Routes/front-routes.php

```diff
@@ -71,17 +71,15 @@
     /**
      * Cart, coupons and checkout.
      */
-    Route::get('mini-cart', [CartController::class, 'getMiniCartDetails'])
-        ->name('velocity.cart.get.details');
+    Route::controller(CartController::class)->name('velocity.cart.')->group(function () {
```

Comment:
Just remove the name prefix and add back the full name to each route. Because during development it is easy to copy the full name rather than in parts and checking. 

Simplification can be good but to some extent.